verify/v1/verify.yaml

```diff
@@ -0,0 +1,808 @@
+        | numberType          | is, is_in            | fixed or mobile. In case of *is_in* operator multiple comma separated values can be provided                             |
+        | operatorCountry     | is, is_in            | country in ISO Alpha-3 format e.g. DEU, GBR. In case of *is_in* operator multiple comma separated values can be provided |
+        | disposableNumber    | is                   | true/false                                                                                                               |
+        | mccMnc              | is, is_in            | MCC and MNC of the operator e.g. 23202. In case of *is_in* operator multiple comma separated values can be provided      |
+        
+        And similarly response are also following the criteria create configuration:
+
+        Response Parameter:
+        
+        - validNumberFormat
+        - numberType
+        - activeNumber
+        - operatorName
+        - OperatorCountry
+        - mcc
+        - mnc
+        - disposableNumber
+
+        In the Response message, parameters will respect the template configuration order.
+        
+        Moreover, following parameters will be always shown in the response:
+        
+        - requestId
+        - verified
+        - reason (only in case of unverified response)
+        
+        However, customers can use those parameters in the response configuration just for manipulating 
+        the order of the Response parameters. 
+
+
+      parameters:
+        - name: criteriaName
+          in: path
+          required: true
+          description: |
+            provide the criteria Name
+          schema:
+            type: string
+            example: "verify-criteria"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateCriteria'
+        required: true
+      security:
+        - apiKey: [ ]
+      responses:
+        200:
+          description: Valid request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UpdateCriteria'
+        400:
+          description: Bad request / Invalid request parameters
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error400'
+        401:
+          description: Missing or Invalid API credentials
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error401'
+        403:
+          description: Permission denied
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error403'
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error404'
+        500:
```

Comment:
after 500 errors, we can add 409 conflict error in Update Criteria (for now it is missing)

409 Conflict
{
  "status": "409",
  "title": "Conflict",
  "detail": ""Template with name <template_name> already exists"",
}